app/views/notifications/_sidebar.html.erb

```diff
@@ -103,6 +103,19 @@
   <% end %>
   <%= menu_separator unless @reasons.empty? %>
 
+  <% if @visiblity %>
+    <% @visiblity.sort_by{|visiblity, count| visiblity.to_s }.each do |visiblity, count| %>
+      <%= filter_link :is_private, visiblity, count do %>
+        <% if visiblity %>
```

Comment:
Visibility will be always true here given `nil.to_s` is true. I think you want to check if visibility is 'true' here.

Reply:
You can see by the result in the page.

![screen shot 2018-08-29 at 14 07 19](https://user-images.githubusercontent.com/47848/44806398-e2993100-ab94-11e8-8306-5965a72088c0.png)

Reply:
Opened as a separate issue here too: https://github.com/octobox/octobox/issues/847